src/sdk/exchange/classes/cross-chain-bridge-supported-chain.ts

```diff
@@ -0,0 +1,30 @@
+import { IsArray, IsBoolean } from 'class-validator';
+import { Type } from 'class-transformer';
+import { BaseClass } from '../../common';
+import { CrossChainBridgeChainCurrency } from './cross-chain-bridge-chain-currency';
+
+export class CrossChainBridgeSupportedChain extends BaseClass<CrossChainBridgeSupportedChain> {
+  chainId: number;
+
+  name: string;
+
+  @IsBoolean()
+  isL1: boolean;
+
+  @IsBoolean()
+  sendingEnabled: boolean;
+
+  icon: string;
+
+  @IsBoolean()
+  receivingEnabled: boolean;
+
+  @Type(() => CrossChainBridgeChainCurrency)
```

Comment:
Is this needed here?
It is no consistent with other classes, for instance:
`cross-chain-bridge-route.ts` and `cross-chain-bridge-route-transaction.ts` have similar properties but decorator